sdk/storage/azure-storage-blob/src/test/java/com/azure/storage/blob/BlobAPITest.groovy

```diff
@@ -682,7 +660,9 @@ class BlobAPITest extends APISpec {
         setup:
         String leaseID = setupBlobLeaseCondition(bu, receivedLeaseID)
 
-        Thread.sleep(16000) // Wait for the lease to expire to ensure we are actually renewing it
+        // If running in live mode wait for the lease to expire to ensure we are actually renewing it
+        sleepIfRecord(16000)
```

Comment:
It seems a long time to run. Can live test finish in 60 mins with the time span like this?
Might need to ignore in live tests

Reply:
I thought the value is second. Ignore the comments here

Reply:
`Thread.sleep` is in milliseconds, running the blob tests in live mode doesn't come close to hitting the 60 minute timeout.